BuildPropertiesPlugin/plugin/build.gradle

```diff
@@ -1,36 +1,29 @@
 buildscript {
+  repositories {
+    jcenter()
+  }
   dependencies {
     classpath 'com.novoda:bintray-release:0.3.4'
   }
 }
 
+repositories {
+  jcenter()
+}
+
 apply plugin: 'groovy'
-apply from: rootProject.file('gradle/publish.gradle')
+apply from: file('../gradle/publish.gradle')
```

Comment:
You're right, this might be a left-over from some previous (failed) experiment. I'll check if that would work 👍